api/tests/fixtures/user_json.json

```diff
@@ -0,0 +1,88 @@
+{
+    "uid": 1,
```

Comment:
I'd rename this file to `osma_api_user_output` to show that this is an output from an external system